AdyenCardScanner/Sources/CardScanner.swift

```diff
@@ -42,4 +43,16 @@ public enum CardScanner: CardScanning {
     public static var isAvailable: Bool {
         cardScannerAssembler.captureDevice != nil
     }
+
+    // MARK: - Private
+
+    // swiftlint:disable line_length
+    private static func assertCameraUsageDescription() {
+        guard let cameraUsageDescription = Bundle.main.object(
```

Comment:
Any particular reason of using a guard here? I believe an if would be easier to read.

if let cameraUsageDescription ..., !cameraUsageDescription.isEmpty { return }
assertionFailure(...)

Reply:
What happens when you don't have the description setup and run it in release mode on the phone? It does not crash right? But does it do anything? Otherwise you might want to return `nil` to not even allow showing the camera screen

Reply:
It won't crash on release mode. At the same time, we don't want to prevent them from using the card scanning feature. This is just a validation to make sure they add the `NSCameraUsageDescription` to their plist file. If they don't, the App Store will take care of reminding them.

Reply:
Ahh I see - should be fine then